docs/readme.ptxt

```diff
@@ -150,8 +150,8 @@ Mind to re-start OpenTTD as it needs to re-read the files.
 4 License
 =========
 
-OpenMSX Music Replacement Set for OpenTTD Copyright (C) 2010 OpenMSX Authors
-(see below or in the source in themes.list) and is licensed under GPL v2.
+OpenMSX for OpenTTD Copyright (C) 2010-2021 OpenMSX Authors (see below or in
```

Comment:
This is the one I think that was intended for you to update - something like:

"OpenMSX music set for OpenTTD: Copyright (C) etc etc"